wgkex/client/__init__.py

```diff
@@ -0,0 +1,12 @@
+from client import app as client
+
+# logging
+import logging
+
+FORMAT = "(%(threadName)-10s) [%(filename)s:%(lineno)s - %(funcName)20s() ] %(message)s"
+logging.basicConfig(level=logging.INFO, format=FORMAT)
+
+
+__author__ = "Annika Wickert <aw@awlnx.space>"
```

Comment:
Let's track authorship, licensing et al in our pyprojects.toml … errrr setup.py!